application/models/SurveyDynamic.php

```diff
@@ -628,6 +631,9 @@ public function search()
         if ($this->completed_filter == "N") {
             $criteria->addCondition('t.submitdate IS NULL');
         }
+        if ($this->completed_filter == "Q") {
+            $criteria->addCondition('tokens.completed = ' . Yii::app()->getDb()->quoteValue('Q'));
```

Comment:
This breaks with error message if token table doesn't exists:
500: Internal Server Error
CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'tokens.completed' in 'where clause'

Check if token table exists should be implemented, something like: if ($this->bHaveToken)